mudlet-mapper.xml

```diff
@@ -6168,6 +6228,12 @@ end
 -- if we can't move, setup a polling timer to prompt walking when we can again.
 -- popular systems that expose balance &amp; equilibrium values can be added here as well, perhaps though a similarly-named function.
 
+if mmp.game == "asteria" then
```

Comment:
I'm not sure this works... The script is ran on startup, at which point the game name is not yet set...

Reply:
You're right, it didn't. I've added a function that'll hook onto the same login event that sets game envids to define this function for Asteria.